all-repos.yaml

```diff
@@ -214,3 +214,4 @@
 - https://github.com/shssoichiro/oxipng
 - https://github.com/datarootsio/databooks
 - https://github.com/standard/standard
+- https://github.com/dannysepler/rm_unneeded_f_str
```

Comment:
- typically ids use dashes rather than underscores: https://github.com/dannysepler/rm_unneeded_f_str/blob/fb3a30dbf8a7eaad60b76bf761972af08f9d6540/.pre-commit-hooks.yaml#L1
- this is wrong: https://github.com/dannysepler/rm_unneeded_f_str/blob/fb3a30dbf8a7eaad60b76bf761972af08f9d6540/.pre-commit-hooks.yaml#L7

Reply:
fixed, thanks!

Reply:
looks like you might also have to fix `rf` and `fr` strings -- glancing at the logic those might be broken

Reply:
good catch! fixed by https://github.com/dannysepler/rm_unneeded_f_str/pull/8